data/microos/butane/config.ign

```diff
@@ -44,7 +44,7 @@
     ],
     "disks": [
       {
-        "device": "/dev/vdc",
+        "device": "/dev/disk/by-id/ata-VMware_Virtual_IDE_Hard_Drive_01000000000000000001",
```

Comment:
Shouldn't this be in sync with config.fcc?

Reply:
This is only for vmware and the device labelling is different. Also since vmware does not use a drive to store ignition/combustion files we use less drives in the tests.

Reply:
I have just kept `config.ign` because it is already used in the test definitions. Ofc, it can be updated accordingly.

Reply:
fixed